style.css

```diff
@@ -0,0 +1,576 @@
+/* Common style: This part is for both home and about page */
+:root {
+  --dark-text: #272a31;
+  --dark-background: #313235;
+  --dark-gray: #797c85;
+  --red: #fa4229;
+  --gray: #f7f7f7;
+  --white: white;
+
+  /* calling: var(--color) */
+}
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  list-style: none;
+  text-decoration: none;
+  color: var(--white);
+  font-family: 'Lato', sans-serif;
+}
+
+body { background-color: var(--gray); }
+
+.main {
+  background-image: url(./images/backgrounds/students.jpg);
```

Comment:
- Fix the size of your image on desktop view:
<img width="1440" alt="Screen Shot 2021-09-13 at 18 50 23" src="https://user-images.githubusercontent.com/22258274/133171610-658dade2-102f-404b-bd10-78f0ee41aeba.png">

Reply:
done